modules/integration/tests-common/clients/store/src/gen/java/org/wso2/am/integration/clients/store/api/v1/dto/CommentDTO.java

```diff
@@ -46,10 +47,77 @@ public class CommentDTO {
         @SerializedName(SERIALIZED_NAME_CREATED_BY)
             private String createdBy;
 
+        public static final String SERIALIZED_NAME_UPDATED_TIME = "updatedTime";
```

Comment:
This `CommentDTO` class has some bad formatting issues (not indented properly). Shall we fix those?

Reply:
Fixed in https://github.com/wso2/product-apim/pull/13178